src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Insights/Add-ArmApplicationInsightsActionGroupArmRoleReceiver.ps1

```diff
@@ -0,0 +1,27 @@
+function Add-ArmApplicationInsightsActionGroupArmRoleReceiver {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    [OutputType("ApplicationInsightsActionGroup")]
+    Param(
+        [PSTypeName("ApplicationInsightsActionGroup")]
+        [Parameter(Mandatory, ValueFromPipeline)]
+        $ActionGroup,
+        [Parameter(Mandatory)]
+        [string]
+        $Name,
+        [Parameter(Mandatory)]
+        [string]
+        $RoleId,
+        [switch]
+        $DisableCommonAlertSchema
```

Comment:
Why not use EnableCommonAlertSchema and use the type bool instead of switch. This way we dont have double negative (not disable)